eisbuk-admin/functions/src/https.ts

```diff
@@ -53,3 +59,126 @@ export const sendEmail = functions
 
     return { ...payload, success: true };
   });
+
+type Protocol = "https" | "http";
+
+/**
+ * Sends SMS message using template data from organizations firestore entry and provided params
+ */
+export const sendSMS = functions
+  .region("europe-west6")
+  .https.onCall(async (payload: Partial<SendSMSPayload>, { auth }) => {
+    // check payload
+    if (!payload) {
+      throw new functions.https.HttpsError(
+        "invalid-argument",
+        HTTPErrors.NoPayload
+      );
+    }
+
+    const { organization, to, message } = payload;
+
+    await checkUser(organization, auth);
+
+    // check payload validity
+    if (!to) {
+      throw new functions.https.HttpsError(
+        "invalid-argument",
+        SendEmailErrors.NoRecipient
```

Comment:
Shouldn't this be `SendSMSErrors.NoRecipient`?

Reply:
Noted, will checking now